cmd/annotations_prep/main.go

```diff
@@ -61,6 +62,30 @@ func (s FeatureStatus) String() string {
 	return "Unknown"
 }
 
+// Source of the {{ .Collection.NameLowercase }}.
+type Source int
```

Comment:
Could there be another source beyond runtime & config? If not a 'runtime' boolean might do?